### PR TITLE
describe one method with a new approach

### PR DESCRIPTION
**Pull request by Raid5594**
Update the documentation to ensure that each method has following sections:
- Description
- Logic
- Permissions
- Params
- Returns
- Events
- Storage
- Errors

### CODE REVIEW

**Review by vpatsenko — APPROVED** [review]

File: bucket/lib.rs [hunk 1]
```diff
@@ -314,14 +314,35 @@ pub mod ddc_bucket {
...
-        /// `cluster_params` is configuration used by clients and nodes. In particular, this describes the semantics of vnodes. See the [data structure of ClusterParams](https://docs.cere.network/ddc/protocols/contract-params-schema)
-        #[ink(message, payable)]
+        /// Params:
+        ///     `cluster_params` is configuration used by clients and nodes. In particular, this describes the semantics of vnodes. See the [data structure of ClusterParams](https://docs.cere.network/ddc/protocols/contract-params-schema)
+        /// 
+        /// Returns:
+        ///     `cluster_id` - id of the newly created cluster.
+        /// 
+        /// Events:
+        ///     ClusterCreated event is triggered, which includes clusterid, managerid, and cluster params
```
Comment (AndreiNavoichyk) [inline bucket/lib.rs:334]: Here can be few events.
Will we use some list? Will we use the list even if we have only one item?

File: bucket/lib.rs [hunk 2]
```diff
@@ -314,14 +314,35 @@ pub mod ddc_bucket {
...
+        /// 
+        /// Storage:
+        ///     Two items are written to storage: 
+        ///         cluster
```
Comment (AndreiNavoichyk) [inline bucket/lib.rs:338]: Let's wrap items with numbered list

File: bucket/lib.rs [hunk 3]
```diff
@@ -314,14 +314,35 @@ pub mod ddc_bucket {
...
 
     impl DdcBucket {
-        /// Create a new cluster and return its `cluster_id`.
-        ///
-        /// The caller will be its first manager.
+        /// Description:
+        ///     Create a new cluster.
         ///
-        /// The cluster is split in a number of vnodes. The vnodes are assigned to the given physical nodes in a round-robin. Only nodes of providers that trust the cluster manager can be used (see `node_trust_manager`). The assignment can be changed with the function `cluster_replace_node`.
+        /// Logic:
```
Comment (AndreiNavoichyk) [inline bucket/lib.rs:320]: Can we wrap it with some formatting here so section names will be more visible in final .md file?

File: bucket/lib.rs [hunk 4]
```diff
@@ -314,14 +314,35 @@ pub mod ddc_bucket {
...
+        ///     ClusterCreated event is triggered, which includes clusterid, managerid, and cluster params
+        /// 
+        /// Storage:
+        ///     Two items are written to storage: 
+        ///         cluster 
+        ///         cluster_params
+        ///     Note: there is currently no methods to delete cluster, hence these items will be stored during the lifetime of the contract
+        /// 
+        /// Errors: 
+        ///     a) if caller is not trusted by nodes included in the cluster
```
Comment (AndreiNavoichyk) [inline bucket/lib.rs:343]: Let's select one type of list we want to use in all lists in documentation. What about numbered?

File: bucket/lib.rs (hunk 1 above)
[the remark this answers is not in the document]
Comment (Raid5594) [reply bucket/lib.rs:334]: Done

Comment (Raid5594) [reply bucket/lib.rs:334]: updated

File: bucket/lib.rs (hunk 2 above)
Comment (Raid5594) [reply bucket/lib.rs:338]: updated

File: bucket/lib.rs (hunk 3 above)
Comment (Raid5594) [reply bucket/lib.rs:320]: updated

File: bucket/lib.rs (hunk 4 above)
Comment (Raid5594) [reply bucket/lib.rs]: using numbers

**Pull request closed without merging**